style.css

```diff
@@ -69,14 +75,385 @@ section {
+.work-section .content-wrapper {
+  padding: 0 16px;
+}
+
+/* project language */
+.work-section .project-lang {
+  list-style: none;
+  display: flex;
+  padding: 0;
+}
+
+.work-section .project1 .project-lang li {
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  margin-right: 12px;
+}
+
+.work-section .project-sample .project-lang li {
+  padding: 10px 12px;
+  margin-right: 12px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+/* images */
+.work-section .project1 .project-img {
+  width: 327px;
+  height: 237px;
+  margin-bottom: 24px;
+}
+
+.work-section .project1 {
+  display: flex;
+}
+
+.work-section .half-btn,
+.contact-section .contact-form button {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+}
+
+.contact-section .contact-form button {
+  width: 131px;
+}
+
+.work-section .project-sample {
+  background-image: url("img/Mask Group.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: white;
+  width: 327px;
+  height: 386px;
+  display: flex;
+}
+
+.work-section .project-sample .full-btn {
+  width: 327px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+}
+
+.work-section .project-sample .full-btn,
+.work-section .half-btn,
+.contact-section .contact-form button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+}
+
+.work-section .project-sample .full-btn:hover,
+.work-section .half-btn:hover,
+.contact-section .contact-form button:hover {
+  color: #f55800;
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.work-section .project-sample .full-btn:active,
+.work-section .half-btn:active,
+.contact-section .contact-form button:active {
+  color: white;
+  background-color: #e05304;
+}
+
+.work-section .project-sample .full-btn:disabled,
+.work-section .half-btn:disabled,
+.contact-section .contact-form button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+.work-section .project-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 12px 0;
+}
+
+.work-section .project-name {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 0;
+}
+
+.work-section .work-section-heading {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  padding: 68px 0;
+  border-bottom: 1px solid black;
+}
+
+/* -----Work section-----end */
+
+/* -------About me------- */
+
+.about-me-section {
+  padding: 0 24px;
+  background-color: #1c1a19;
+  color: white;
+  background-image: url("img/Frame.png");
+  background-repeat: no-repeat;
+  background-position: right 70vh;
+}
+
+.about-me-section .grid-wrapper {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: 100vh;
+  row-gap: 24px;
+  padding: 80px 0;
+}
```

Comment:
- [OPTIONAL] KIndly make sure to align the `About myself` section contents using CSS `flexbox` in order to meet the project requirements.

    ### PROJECT REQUIREMENT

     ![image](https://user-images.githubusercontent.com/87197412/227998698-7b33f850-47bf-40b7-a5a5-9508247c236d.png)